src/components/DecisioningEngine/createContextProvider.js

```diff
@@ -9,8 +9,39 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
+import { extractBrowserDetails } from "../../utils/extractBrowserDetails";
+
 export default ({ eventRegistry }) => {
-  const globalContext = {}; // holder of global context like current time/date, browser name/version, day of week, scroll position, time on page, etc
+  const { browserName, browserVersion } = extractBrowserDetails();
+  const timestamp = new Date();
+  const globalContext = {
+    timePageLoaded: timestamp.getTime(),
+    datePageLoaded: timestamp.getDate(),
+    dayPageLoaded: timestamp.getDay(),
+    currentDate: timestamp.getDate(),
+    currentTime: timestamp.getTime(),
+    currentDay: timestamp.getDay(),
+    scrollPosition: window.scrollY,
+    browserDetails: {
+      browserName,
+      browserVersion
+    },
+    pageContext: {
+      pageName: document.title,
+      pageURL: window.location.href,
+      pageReferrer: document.referrer,
+      pageHeight: window.innerHeight,
+      pageWidth: window.innerWidth
+    }
+  };
+
+  setInterval(() => {
```

Comment:
window context should also be updated on each `getContext` call so that up to date window size and scroll positions are used.

But stuff like page, referringPage, browser name/version, etc should only be set once when `createContextProvider` is initialized